src/core/Track.cpp

```diff
@@ -2612,7 +2612,11 @@ void TrackView::mousePressEvent( QMouseEvent * _me )
 	}
 
 
-	if( m_trackContainerView->allowRubberband() == true )
+	int widgetTotal = ConfigManager::inst()->value( "ui",
```

Comment:
This is the width of the left part of the track view, with all the buttons and leds and stuff right? Good job! :-)